app/src/main/java/com/example/stretchy/activity/MainActivity.kt

```diff
@@ -65,6 +79,61 @@ class MainActivity : ComponentActivity() {
             )
         }
     }
+    @Composable
+    fun BottomNavigationBar() {
+        val navController = rememberNavController()
+
+        val screens = listOf(
+            BottomNavScreen("Screen 1", Icons.Default.Home, "screen1"),
+            BottomNavScreen("Screen 2", Icons.Default.Search, "screen2"),
+            BottomNavScreen("Screen 3", Icons.Default.Person, "screen3"),
+        )
```

Comment:
Screens should have their name set, those should be in strings. I propose 
`Stretching`
`Training`
`Meta Training`